pkg/template/helm_template_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package template
```

Comment:
should be in a template_test package so that we do not use template's privates.

Reply:
In that case I'll not be able to unit test the un-exported `helmTemplateCmdLookup` function. 

I can see two ways a) To export it? b) Do not unit test the unexported method and instead do test `(t *HelmTemplate) **TemplateDir(...)`? 

What would you suggest? I think ideally we should have b) but that might require way more work in terms of decoupling the code from the binary and leveraging interfaces instead and I am not sure if it is in the scope of this specific PR.

Reply:
i would recommend exporting this function and associated return struct. (also might be good to rename this function to NewHelmTemplateCmdArgs since it's really acting as a constructor for HelmTemplateCmdArgs)

Reply:
Thanks!